test/webhook_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+sed -ri "s|'(quay.io/reactiveops/polaris:).+'|'\1${CIRCLE_SHA1}'|" ./deploy/webhook.yaml
+kubectl apply &> /dev/null -f ./deploy/webhook.yaml
+sleep 15
+
+kubectl apply &> /dev/null -f test/correctconfig.yaml
+if [ $? -eq 0 ]; then
+    VAR1="pass"
+    echo pass 
+else
+    echo "Test Failed: Polaris prevented a deployment with no configuration issues." 
+fi
+
+kubectl apply &> /dev/null -f test/incorrectconfig.yaml
+if [ $? -ne 0 ]; then
+    VAR2="pass"
+    echo pass 
+else
+    echo "Test Failed: Polaris should have prevented this deployment due to configuration problems."
+fi
+
+if [ "$VAR1" == "pass" -a "$VAR2" == "pass" ]; then
```

Comment:
Nit: As future people augment this to add more test cases, adding more of these `and ...` type blocks may become tedious. An alternative is to define a basic framework for your script by defining a variable at the top like `ALL_TESTS_PASSED=true` and then as you run the tests you can call a function if it fails to set `ALL_TESTS_PASSED=false`. Then at the end of the script you can do a simple check:

```shell
if $ALL_TESTS_PASSED; then
  echo "yey!"
else
  echo "booo tests failed"
  exit 1
fi
```